tarantool/connection.py

```diff
@@ -1269,6 +1291,18 @@ def flush_schema(self):
         self.schema.flush()
         self.load_schema()
 
+    def _fetch_schema_support_check(self):
+        """
+        Checks the fetch_schema opt of the connection is enabled.
```

Comment:
Description is outdated

Reply:
Description updated and func renamed to `_schemafull_connection_check`